pkg/dyninst/symdb/cli/main.go

```diff
@@ -49,13 +49,16 @@ The symbols from the specified binary will be extracted and printed to stdout
 		logLevel = "info"
 	}
 	log.SetupLogger(log.Default(), logLevel)
+	defer log.Flush()
```

Comment:
you know this won't get called on `os.Exit`, right?

Reply:
that's why I call it again below.

all the logic is already part of `run`, but I think this `defer` doesn't hurt